assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -2812,8 +2812,10 @@ private String build_list_assignments_context(VelocityPortlet portlet, Context c
         }
 
         List<Assignment> assignments = prepPage(state);
+        int allAssignmentSize = assignments.size();
```

Comment:
It looks like there is a variable already `hasAssignments` so there is likely no need to add another?

Reply:
Correct. I've already changed it. Thank you so much.